contracts/hackatom/src/contract.rs

```diff
@@ -273,4 +317,34 @@ mod tests {
             }
         );
     }
+
+    #[test]
+    #[should_panic]
+    fn handle_panic() {
+        let mut deps = dependencies(20);
+
+        // initialize the store
+        let verifier = HumanAddr(String::from("verifies"));
+        let beneficiary = HumanAddr(String::from("benefits"));
+        let creator = HumanAddr(String::from("creator"));
+
+        let init_msg = InitMsg {
+            verifier: verifier.clone(),
+            beneficiary: beneficiary.clone(),
+        };
+        let init_params = mock_params(
+            &deps.api,
+            creator.as_str(),
+            &coin("1000", "earth"),
+            &coin("1000", "earth"),
+        );
+        let init_res = init(&mut deps, init_params, init_msg).unwrap();
+        assert_eq!(0, init_res.messages.len());
+
+        // beneficiary can release it
```

Comment:
This comment looks like a copy/paste mistake

Reply:
Yup. It was